docs/content/en/docs/Coding Milestones/PoC2023q1/edge-syncer.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Edge Syncer"
+date: 2023-03-22
+weight: 4
+description: >
+---
+
+{{% pageinfo %}}
+Edge syncer is a new syncer working with mailbox workspace in edge-mc. 
+{{% /pageinfo %}}
+
+![edge-syncer drawio](./images/edge-syncer-overview.png)
+
+#### Registering Edge Syncer on a p-cluster
+
+Edge-syncer can be deployed on p-cluster easily by the following steps.
+1. Create SyncTarget and Location
+  - Mailbox controller creates mailbox workspace automatically. 
+  - Get mailbox workspace id
+2. Use command to register edge-syncer
+   ```console
+   kubectl ws <mb-ws name>
+   kcp workload edge-sync <EM Sync Target name> --image <EM Syncer Image> -o edge-syncer.yaml
+   ```
+3. Deploy edge-syncer on p-cluster.
+
+![edge-syncer boot](./images/edge-syncer-boot.png)
+
+#### Deploy workload objects from edge-mc to p-cluster
+
+To deploy resources to p-clusters, create the following in workload management workspace
+- workload objects
+  - Some objects are denatured if needed.
+  - Other objects are as it is
+- APIExport/API Schema corresponding to CRD
+  - TBD: Conversion from CRD to APIExport/APISchema could be automated by using MutatingAdmissionWebhook on workload management workspace.
+- EdgeSyncConfig (This is going to be replaced with EdgePlacement spec) 
+
+![edge-syncer deploy](./images/edge-syncer-deploy.png)
+
+After this, Edge-mc will put the following in the mailbox workspace.
+- Workload objects (both denatured one and not-denatured one)
+- API Binding (to API Export on the workload management workspace)
+- EdgeSyncConfig CR
+
+**TODO**: This is something we should clarify..e.g. which existing controller(s) in edge-mc will cover, or just create a new controller to handle uncovered one. 
+
+#### EdgeSyncConfig
+- The example is [here](https://gist.github.com/yuji-watanabe-jp/dfa9cdf35b06f612818446c2c3eb7969).
+- The CR here is used from edge syncer. 
+- The CR is placed at mb-ws to define
+  - object selector
+  - need of renaturing
+  - need of status upsync for downsyncing
+  - need of delete propagation for downsyncing
```

Comment:
Is there a reason to not have this always on?

Reply:
just for reducing the unnecessary returns and propagation in some cases.

Reply:
Can you give examples?

Reply:
if CRD is delivered to p-cluster, and many CRs are created. Even if the CRD is deleted at mailbox, I may want to keep CRD on p-cluster side, because many CRs cannot live without the CRD.